src/Files.App/Views/SettingsPages/Tags.xaml

```diff
@@ -106,6 +106,40 @@
 								HorizontalAlignment="Right"
 								Orientation="Horizontal"
 								Spacing="12">
+								<!--  Invalid Name Warning  -->
+								<Grid
```

Comment:
Why isn't this a regular info bar?

Reply:
The padding on the regular InfoBar was too large and I couldn't find a simple way to reduce it

Reply:
I'll find the resource for it, can you switch to it?

Reply:
Fix'd